src/app/src/components/Layers/ParcelLayer.js

```diff
@@ -0,0 +1,69 @@
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { DynamicMapLayer } from 'esri-leaflet';
+
+import { useMapLayer } from '../../hooks';
+import { PARCELS_LAYER_URL, DATA_LAYERS } from '../../constants';
+
+const PARCELS_LAYER_STYLE = [
+    {
+        id: 0,
+        source: {
+            type: 'mapLayer',
+            mapLayerId: 0,
+        },
+        drawingInfo: {
+            renderer: {
+                type: 'simple',
+                symbol: {
+                    type: 'esriSMS',
+                    style: 'esriSMSDiamond',
+                    color: [183, 121, 31, 255], // var(--chakra-colors-yellow-600)
+                },
+            },
+        },
+    },
+    {
+        id: 1,
+        source: {
+            type: 'mapLayer',
+            mapLayerId: 1,
+        },
+        drawingInfo: {
+            renderer: {
+                type: 'simple',
+                symbol: {
+                    type: 'esriSFS',
+                    style: 'esriSFSSolid',
+                    color: [0, 0, 0, 0], // No fill
+                    outline: {
+                        type: 'esriSLS',
+                        style: 'esriSLSSolid',
+                        color: [183, 121, 31, 255], // var(--chakra-colors-yellow-600)
+                        width: 1.0,
+                    },
+                },
+            },
+        },
+    },
+];
+
+export default function ParcelLayer() {
+    const layers = useSelector(state => state.map.layers);
+    const showLayer = useMemo(() => layers.includes('PARCELS'), [layers]);
+
+    return showLayer ? <RenderEsriLayer /> : null;
+}
```

Comment:
I created a hook called `useMapVisibility` in https://github.com/azavea/iow-boundary-tool/pull/46 that encapsulates some of this functionality, but perhaps it makes more sense to introduce it here since I'm no longer using it in #46.

```js
export function useLayerVisibility(layer) {
    return useSelector(state => state.map.layers).includes(layer);
}
```

Reply:
Nice! Should this be wrapped in `useMemo` as the original is?

Reply:
like:

```js
export function useLayerVisibility(layer) {
    const layers = useSelector(state => state.map.layers)
    const showLayer = useMemo(() => layers.includes('PARCELS'), [layers]);

    return showLayer;
}
```

or does `useMemo` only make sense in the context of a React Component?

Reply:
I'm not sure. Since the returned value is a boolean which is a primitive type, I think it's already memoized (i.e. it won't cause re-render when passed as a prop and it's value hasn't chaged, but it has been recalculated). 

I figured we could remove the hook and add it back if there are noticable performance issues.

Reply:
Naïve implementation (without `useMemo`) in 44bbd4b